frontend/awx/resources/inventories/inventoryGroup/GroupPageAdd.tsx

```diff
@@ -0,0 +1,54 @@
+import { useTranslation } from 'react-i18next';
```

Comment:
The name of the file...  Generally our conventions has been to name the file that wraps the form fields something like InventoryGroupForm.  Inside it we export 2 functions 1 for add and another for edit  (See ProjectForm for an example).  

While this request is super nitpicky I think it would improve developer experience if we can keep our file name in line with the conventions.